test/Logo.spec.js

```diff
@@ -0,0 +1,13 @@
+import { Logo } from '../src/app/layouts/Logo';
+
+describe('Logo', () => {
+  it('Test logo with "logo" exist', () => {
```

Comment:
Spróbuj to układać bardziej w "zdania" i żeby testy mówiły co dokładnie sprawdzają. Czyli np. 

NIE: `Check if the link is correct`
TAK: `link should navigate to .` - tzn. opis co znaczy to "correct" 

Tak samo pierwszy np. `should have id with value = "logo"`

No i potem czytasz to jako zdanie (razem describe z it) np: 'Logo should have id with value = "logo"' i jest ok :) Test to jest Twoje założenie co powienien (should) kod, potem się okazuje jak go puszczasz czy tak jest.

Reply:
Poprawię to w wolnej chwili. (chyba już jutro).
Dziękuję za uwagę! Za mało opisów jeszcze widziałem w życiu!